services/QuillConnect/app/components/concepts/concept.jsx

```diff
@@ -31,14 +31,15 @@ const Concept = React.createClass({
   },
 
   questionsForConcept: function () {
-    var questionsCollection = hashToCollection(this.props.questions.data)
-    return _.where(questionsCollection, {conceptID: this.props.params.conceptID})
+    const questionsCollection = hashToCollection(this.props.questions.data)
+    return questionsCollection.filter(q => q.conceptID === this.props.params.conceptID && q.flag !== 'archived')
   },
 
   renderQuestionsForConcept: function () {
     var questionsForConcept = this.questionsForConcept()
     var listItems = questionsForConcept.map((question) => {
-      return (<li key={question.key}><Link to={'/admin/questions/' + question.key}>{question.prompt.replace(/(<([^>]+)>)/ig, "").replace(/&nbsp;/ig, "")}</Link></li>)
+      const archivedTag = question.flag === 'archived' ? <strong>ARCHIVED - </strong> : ''
+      return (<li key={question.key}><Link to={'/admin/questions/' + question.key + '/responses'}>{archivedTag}{question.prompt.replace(/(<([^>]+)>)/ig, "").replace(/&nbsp;/ig, "")}</Link></li>)
```

Comment:
I wonder if this line hasn't become complicated enough that it's worth breaking it out into multiple lines.
```typescript
return (
  <li key={question.key}>
    <Link to={'/admin/questions/' + question.key + '/responses'}>
      {archivedTag}
      {questions.prompt.replace(/(<([^>]+)>)/ig, "").replace(/&nbsp;/ig, "")
    </Link>
  </li>
)
```
I don't think it's really necessary to do that here in this PR, but I think that this is one of those cases where the verbosity (in the sense of using extra lines) enhances clarity a lot.

Reply:
That's fair, thank you! I'll try to be more attentive about that.